src/tooltip/readme.tsx

```diff
@@ -138,6 +156,26 @@ export default function Readme() {
         </>
       </DocsExample>
 
+      <DocsExample label="Alignment">
+        <>
+          <Tooltip overlay={'Align start'} align="start">
```

Comment:
[nit] The outer `{` is not necessary? So you could write these as `overlay="Align start"`